troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js

```diff
@@ -16,11 +16,11 @@ define(function (require) {
 
     onDelete: function () {
       var project = this.props.project,
-        link = this.props.link;
+        external_link = this.props.link;
 
       modals.ExternalLinkModals.destroy({
-        link: this.props.link,
-        project: this.props.project
+        project,
+        external_link
```

Comment:
Just FYI, with ES2015 we can do this now. 
`{ name: name } === { name } // true`